diagnostic.md

```diff
@@ -13,57 +16,61 @@ Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+```js
+$('li');
+// OR
+$( 'ol' ).children();
+```
 
 ## Question 3
 
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+```js
+$('.listItem').closest('form');
```

Comment:
`.closest()` traverses upwards and won't find `<form>` since it's a sibling of `<ol>`.  It will go from a `<li>` to `<ol>` to `<body>` to `<html>`.